createuser.php

```diff
@@ -1,5 +1,51 @@
 <?php
-require_once('php/connection.php');
-function validateUser($email, $password) {
-    
-}
\ No newline at end of file
+
+session_start();
+if(isset($_SESSION["auth"])){  
+
+if($_SESSION["auth"] == "2")  
+{
+ 
+}
+else
+{
+    header("Location: login.php?e=4");
+ die("You are not Signed in.");  }  
+}
+
+$sql = "UPDATE
+`users`
+SET
+`firstname` = '$f_name',
+`lastname` = '$l_name',
+`email` = '$email',
+`password` = '$password'
+WHERE
+`users`.`userid` = $uid";
```

Comment:
This entire SQL statement is wrong.
Firstly, 'update' modifies an existing user. You want 'insert' to create a new user. __BUT DON'T DO THIS__
There is already a function in SQL called 'CreateUser' that will create a user with some given parameters. Your SQL should look something like `CALL CreateUser(:userID, :firstname, :lastname, :email, :password);`